doc/articles/features/uno-cupertino.md

```diff
@@ -1,59 +1,4 @@
 # Uno.Cupertino
```

Comment:
why not just drop the page entirely, and redirect all links pointing here to the new place?

Reply:
We can, I'm assuming that'd mean changes on the server would be needed right?

Reply:
any change, structural or content-wise, would have required that

Reply:
There's no "server-side" here, everything needs to be implemented here. If docfx has features to perform redirects, let's use that.

Reply:
i mean redirect, as in updating all the links in every *.md
not the 301 server redirect